app/models/firm.rb

```diff
@@ -25,6 +25,7 @@ class Firm < ActiveRecord::Base
   belongs_to :parent, class_name: 'Firm'
 
   has_many :advisers, dependent: :destroy
+  has_many :offices, -> { order created_at: :asc }
```

Comment:
And also it's listed in the acceptance criteria on the card: https://trello.com/c/5ab7WP1I